python/tchannel/scheme.py

```diff
@@ -41,10 +62,18 @@ def __init__(self):
     def type(self):
         return self._type
 
-    def deserialize(self, obj):
+    def deserialize_header(self, obj):
+        assert obj
```

Comment:
For the edge cases like None or empty, I am planning to include them in the error handling tasks. At that time, I'll list all the possible cases in tchannel with corresponding responses of tchannel.